_studio/mfx_lib/scheduler/include/mfx_scheduler_core.h

```diff
@@ -32,9 +32,6 @@
 // synchronization stuff
 #include <vm_time.h>
 
-#include <umc_semaphore.h>
-#include <umc_event.h>
-
 #include <vector>
```

Comment:
Please, break into 2 commits where you remove instances one by one.

Reply:
Done